interface/globals.php

```diff
@@ -192,6 +192,15 @@ function undoMagicQuotes($array, $topLevel=true) {
 // Include the version file
 require_once (dirname(__FILE__) . "/../version.php");
 
+// The logging level for common/logging/logger.php
+$GLOBALS["log_level"] = "DEBUG"; // (value can be TRACE, DEBUG, INFO, WARN, or ERROR)
+
+// Should Doctrine run in "dev mode"?
+$GLOBALS["doctrine_dev_mode"] = false;
```

Comment:
Doctrine makes you supply this boolean. It's called out in this section: http://docs.doctrine-project.org/projects/doctrine-orm/en/latest/reference/configuration.html#obtaining-an-entitymanager

It's probably never going to be set as `true`, but I am big on externalizing configuration where possible. *shrugs*.